core/src/main/java/edu/softserve/zoo/model/Role.java

```diff
@@ -38,7 +38,17 @@ public int hashCode() {
     }
 
     public enum Type {
-        VISITOR, EMPLOYEE, MANAGER, ADMIN;
+        VISITOR("ROLE_VISITOR"), EMPLOYEE("ROLE_EMPLOYEE"), MANAGER("ROLE_MANAGER"), ADMIN("ROLE_ADMIN");
+
```

Comment:
Don't add new fields. Leave it as-is and add appropriate logic in userloadservice